pkgpanda/__init__.py

```diff
@@ -612,8 +612,12 @@ def add_user(self, username, groupname):
                                   "automatic user addition is disabled".format(username))
 
         # Add the user:
+        useradd_cmd = 'useradd'
+        if os.path.exists('/usr/sbin/useradd'):
+            useradd_cmd = '/usr/sbin/useradd'
+
         add_user_cmd = [
-            'useradd',
+            useradd_cmd,
```

Comment:
Why is this change necessary? Is there a competing `useradd` earlier in the `PATH` that we need to avoid?

Reply:
I'll add a comment. This is because the PATH used by pkgpanda doesn't include `/usr/sbin` which is the location on Ubuntu. On Azure, it's symlinked into `/usr/bin`, but I was attempting to get this working on stock Ubuntu without customization.